drivers/input/input_ft5336.c

```diff
@@ -88,24 +93,39 @@ static int ft5336_process(const struct device *dev)
 	}
 
 	points = (points >> TOUCH_POINTS_POS) & TOUCH_POINTS_MSK;
-	if (points != 0U && points != 1U) {
-		return 0;
-	}
-
-	/* obtain first point X, Y coordinates and event from:
-	 * REG_P1_XH, REG_P1_XL, REG_P1_YH, REG_P1_YL.
-	 */
-	r = i2c_burst_read_dt(&config->bus, REG_P1_XH, coords, sizeof(coords));
-	if (r < 0) {
-		return r;
-	}
+	if (points != 0) {
+		/* Any number of touches still counts as one touch.
+		 * All touch points except the first are ignored.
+		 * Obtain first point X, Y coordinates from:
+		 * REG_P1_XH, REG_P1_XL, REG_P1_YH, REG_P1_YL.
+		 * We ignore the Event Flag because Zephyr only
+		 * cares about pressed / not pressed and not
+		 * press down / lift up
+		 */
```

Comment:
You have 80 columns, use them :-)

Reply:
also, you can run the compliance check automation locally, for a single patch it's just `./scripts/ci/check_compliance.py`, saves you from waiting for CI to do it for you

Reply:
Doesn't seem to work under Windows. I'll fight with it later.